VMEncryption/main/check_util.py

```diff
@@ -107,6 +115,23 @@ def check_kv_id(self, test_kv_id, message):
             raise Exception('\n' + message + '\nActual: ' + test_kv_id + '\nExpected: ' + expected + "\n")
         return
 
+    def check_mhsm_id(self, test_mhsm_id, message):
```

Comment:
You can try to replicate the `check_kv_id` function to check_mhsm_id` function with few updates. Both essentially do the same thing with a little difference. Good to have similar implementation.

Reply:
I think I missed a commit here, this looks unfinished. I will revisit these checks and implement tests, I must have forgotten a save